MAVProxy/modules/mavproxy_misseditor/mission_editor.py

```diff
@@ -229,6 +232,9 @@ def process_mavlink_packet(self, m):
 
         # if you add processing for an mtype here, remember to add it
         # to mavlink_packet, above
+        if (getattr(m, 'mission_type', None) is not None and
+            m.mission_type != mavutil.mavlink.MAV_MISSION_TYPE_MISSION):
+            return
```

Comment:
Is this one really needed? I mean, if it isn't put in the queue (given the new code in `mavlink_packet`), then it doesn't get here, right?